api/swagger-spec/resourceListing.json

```diff
@@ -2,6 +2,10 @@
   "swaggerVersion": "1.2",
   "apis": [
    {
+    "path": "/logs",
```

Comment:
There should be a new logs.json file. You will need to update update-swagger-spec.sh to include it.